packages/component/src/Utils/Timer.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+export default class Timer extends React.Component {
+  componentDidMount() {
+    this.schedule(this.props.at);
+  }
+
+  componentWillReceiveProps(nextProps) {
```

Comment:
deprecated.

Reply:
here and elsewhere... I believe this method is now deprecated and should be replaced.

Reply:
Got no deprecation warnings on build. But agree we need to move it forward.